components/engine/src/rocks/util/event_listener.rs

```diff
@@ -64,6 +66,34 @@ impl engine_rocksdb::EventListener for EventListener {
             .inc();
     }
 
+    fn on_background_error(&self, reason: DBBackgroundErrorReason, result: Result<(), String>) {
+        assert!(result.is_err());
+        if let Err(err) = result {
+            STORE_ENGINE_EVENT_COUNTER_VEC
```

Comment:
em, I guess this metric has no effect because we will panic soon.

Reply:
seems we also bump metrics before panic in other places, e.g. https://github.com/tikv/tikv/blob/master/src/storage/kv/mod.rs#L649

Reply:
you can see that the following has condition check before panic, not always panic.

but here we will panic all the time. so the metric is meaningless.

Reply:
+1.  Maybe we could sleep a little time to wait prometheus reporting counter to server?

Reply:
prometheus pull is a 15s cycle, which is too long. let's remove the metrics.